app/helpers/assignments_helper.rb

```diff
@@ -2,11 +2,13 @@ module AssignmentsHelper
 
   def add_assignment_file_link(name, form)
     link_to_function name do |page|
-      assignment_file = render(partial: 'assignment_file', locals: {form: form, assignment_file: AssignmentFile.new})
+        assignment_file = render(partial: 'assignment_file', locals: {form: form, assignment_file: AssignmentFile.new})
```

Comment:
Line is too long. [119/80]<br>Use 2 (not 4) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.